### PR TITLE
added composer.json, added instructions in README.md

### PR DESCRIPTION
**Pull request by vardumper**
added composer.json and extended install instructions

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,7 @@ node_modules/
 tests/
 .nvmrc
 .prettierrc.js
-*.DS_Store
\ No newline at end of file
+*.DS_Store
+/.settings/
```
Comment (vardumper) [inline .gitignore:6]: did not know how to prevent this update, it's really just some temp files my IDE copies into all projects I work on.

**Comment by lucaslarroche** [comment]
Hi @vardumper, 

I'm not 100% against it, but I wonder if it's really necessary.
Since Pico is not in PHP, and composer is a dependency management tool for PHP, I wonder how it can be used for Pico users.

I would like more feedback.

**Comment by lucaslarroche** [comment]
@yoanbernabeu, 

Sorry to ping you here 🙂. I could use your advice.
I noticed your [video](https://www.youtube.com/watch?v=8siyYpMv_KY&ab_channel=YoanDev) and your [integration](https://github.com/yoanbernabeu/Pico.css-avec-Symfony-et-Webpack-Encore). 
Merci 🇫🇷

Do you think it is important or necessary to integrate a composer config file in Pico?

**Comment by vardumper** [comment]
Independently of what Bootstrap does... 

... this is really just for the convenience of installing pico whatever way someone wats to install it.
In my (PHP) project, I use `scssphp/scssphp` to compile sass files, so after installing picots with composer I just added `@import '../vendor/picocss/pico/scss/pico.scss';`

this way I don't need to .gitignore a /node_modules folder, nor do I need to install npm, or manage a packages.json. Makes my deploy process leaner.

**Comment by yoanbernabeu** [comment]
> @yoanbernabeu,
> 
> Sorry to ping you here slightly_smiling_face. I could use your advice. I noticed your [video](https://www.youtube.com/watch?v=8siyYpMv_KY&ab_channel=YoanDev) and your [integration](https://github.com/yoanbernabeu/Pico.css-avec-Symfony-et-Webpack-Encore). Merci fr
> 
> Do you think it is important or necessary to integrate a composer config file in Pico?

Hello and thank you for the mention.
I just saw your messages... too many emails :)

I don't think it's necessary to add a package for the tool, and it would require additional maintenance time IMHO.

**Comment by vardumper** [comment]
For me this topic is solved. Feel free to close the ticket. 
Roundup for anybody interested or stumbling over this PR: https://erikpoehler.com/2022/02/27/build-watch-sass-files-with-php/

**Comment by praneetloke** [comment]
> I don't think it's necessary to add a package for the tool, and it would require additional maintenance time IMHO.

To be fair, `composer.json` seems to simply provide metadata that helps for a smoother installation for PHP users. No? I am wondering what the additional maintenance is for having that file in the repo? By contrast looking at twbs' `composer.json` link that @lucaslarroche posted, it seems that the file had very few modifications in its lifetime and the last time it was modified was back in 2019, so I'd say it is not much maintenance for something as large as Bootstrap. However, I am not a seasoned PHP dev so I don't know the ecosystem well enough but I am checking out using Laravel for something so this is slightly relevant for me.

To me this PR seems like a simple yet useful one that helps PHP users install this as a package without requiring local modifications as described by @vardumper in #146.

**Comment by prikeshsavla** [comment]
I like the idea to support more package managers, which allows a smoother addition to other language frameworks.

**Comment by lucaslarroche** [comment]
Thank you, @yoanbernabeu, @praneetloke, @prikeshsavla.
Your comments are helpful. I will probably merge this PR.
It won't be in the [next release](https://github.com/picocss/pico/pull/155) as I want to set up a small PHP project as a test and create a Packagist account to be correctly listed.

**Pull request merged**